server/game/core/Player.js

```diff
@@ -67,7 +67,9 @@ class Player extends GameObject {
 
         this.playableLocations = [
             new PlayableLocation(PlayType.PlayFromHand, this, Location.Hand),
-            new PlayableLocation(PlayType.Smuggle, this, Location.Resource)
+            new PlayableLocation(PlayType.Smuggle, this, Location.Resource),
+            new PlayableLocation(PlayType.PlayFromOutOfPlay, this, Location.Deck),
+            new PlayableLocation(PlayType.PlayFromOutOfPlay, this, Location.Discard),
```

Comment:
@AMMayberry1 Do we want two records for one play type like this, or do we want to enhance the location to accept an array?